presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -212,14 +231,14 @@ public JdbcOutputTableHandle beginInsertTable(ConnectorSession session, JdbcTabl
     public void finishInsertTable(ConnectorSession session, JdbcOutputTableHandle handle)
     {
         delegate.finishInsertTable(session, handle);
-        invalidateTablesCaches();
+        invalidateTablesCaches(toTableHandle(handle));
```

Comment:
Actually, I don't know why why are invalidating here.

Reply:
Idk either. I'll remove that.